baseuser/models.py

```diff
@@ -31,11 +31,12 @@ def __str__(self):
 
 class UserProfile(models.Model):
     base_user = models.OneToOneField(BaseUsers, on_delete=models.CASCADE,
+
                                      related_name='profile')
     current_company = models.ForeignKey(Company, null=True, blank=True,
                                         on_delete=models.CASCADE,
                                         related_name='current_company')
-    past_companies = models.ManyToManyField(Company, null=True, blank=True,
+    past_companies = models.ManyToManyField(Company, blank=True,
                                             related_name='past_companies')
```

Comment:
@petertheprofessional @divyaChandran10 , what is the need of the related name in the Many to many field?